### PR TITLE
Background fix

### PR DESCRIPTION
**Pull request by IlyaRucavitcyn**
Suggest to merge the tree into master

### CODE REVIEW

File: iconFont/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Font license info
```
Comment (shemerey) [inline iconFont/LICENSE.txt]: clean up this file, it looks like a junk

File: css/style.css [hunk 2]
```diff
@@ -157,38 +181,66 @@ section{
...
+  content: '\e800';
+} /* 'Ã®Â â‚¬' */
+
+.icon-dollar:before {
+  content: '\e801';
+} /* 'Ã®Â Â' */
+
+.icon-rouble:before {
+  content: '\e802';
+} /* 'Ã®Â â€š' */
```
Comment (shemerey) [inline css/style.css]: clean up all unnecessary comments

File: iconFont/README.txt [hunk 3]
```diff
@@ -0,0 +1,75 @@
+This webfont is generated by http://fontello.com open source project.
```
Comment (shemerey) [inline iconFont/README.txt]: clean up this file, it looks like a junk

File: iconFont/config.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
+{
```
Comment (shemerey) [inline iconFont/config.json]: why you need this file ?

File: iconFont/demo.html [hunk 5]
```diff
@@ -0,0 +1,312 @@
+<!DOCTYPE html>
```
Comment (shemerey) [inline iconFont/demo.html]: clean up this file, it looks like a junk

**Comment by IlyaRucavitcyn** [comment]
Cleaned Up!

**Comment by shemerey** [comment]
LGTM

**Pull request merged**